bin/ringling-cli

```diff
@@ -58,14 +84,58 @@ if __name__ == "__main__":
 
     if args.object_type == "project":
```

Comment:
You can clean a lot of this up using [subparsers](https://docs.python.org/3/library/argparse.html#sub-commands).  Parameters that are not needed for every command can be attached to individual subparsers and then the library can handle checking  for required arguments..  You can see examples of their use [here](https://github.com/nowling-lab/asaph/blob/main/bin/asaph_genotype#L334).